ygen/codegen_test.go

```diff
@@ -1563,16 +1563,6 @@ func TestGetDirectoriesAndLeafTypes(t *testing.T) {
 				},
 				Path: []string{"", "enum-module", "parent", "child"},
 			},
-			"/enum-module/a-lists/a-list": {
```

Comment:
This is because I added `config false;` to testdata/modules/enum-module.yang in order to make its leafref make sense, since it's referring to a state value. A config leafref can't refer to a state node.

Once this is added, this list is now excluded since it's state-only.